packages/core/src/components/CardListItem/CardListItem.tsx

```diff
@@ -45,14 +16,31 @@ export type CardListItemProps = {
 export default function CardListItem(props: CardListItemProps) {
   const { children, selected, onSelect, loading, disabled, noPadding = false, ...rest } = props;
 
-  const content = noPadding ? (
-    <StyledCardContentNoPadding>{children}</StyledCardContentNoPadding>
-  ) : (
-    <StyledCardContent>{children}</StyledCardContent>
+  const content = (
+    <CardContent sx={{ marginBottom: (theme) => (noPadding ? 0 : `${theme.spacing(2)} !important`) }}>
+      {children}
```

Comment:
Previously, `StyledCardContent` and `StyledCardContentNoPadding` set padding/paddingBottom, not marginBottom. 

This is affecting the layout of the wallet keys, and may also affect other areas of the UI (though I'm unsure where).

The padding should be 0 in this case:

Wrong:
![image](https://user-images.githubusercontent.com/339312/228082356-8292d07c-8df7-427e-8a20-5e0088705f20.png)

Correct:
![image](https://user-images.githubusercontent.com/339312/228085945-db744924-4e87-4ec9-8520-88bbd568924b.png)